app/controllers/player_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PlayerController < ApplicationController
+  include SlideUtil
+  before_action :set_slide
+
+  def show
+    # increment only when the player is embedded in other site...
+    unless params.key?(:inside) && params[:inside] == '1'
+      @slide.increment(:embedded_view).increment(:total_view).save
+    end
+    @start_position = slide_position
+    s = render_to_string layout: 'plain', collection: @slide
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 's' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->